open-machine-learning-jupyter-book/deep-learning/rnn/bi-rnn.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
<p>The path of imdb.npz is in your local environment. You need to put it in the COS cloud disk and download it locally.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/749/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>